org.adempiere.base/src/org/compiere/print/DataEngine.java

```diff
@@ -171,9 +171,6 @@ public PrintData getPrintData (Properties ctx, MPrintFormat format, MQuery query
 		if (format == null)
 			throw new IllegalStateException ("No print format");
 
-		 if (format.getJasperProcess_ID() > 0)
-			 return null;
-
```

Comment:
this is a dangerous change, you don't want to call getPrintDataInfo and loadPrintData for Jasper Report print format.

Reply:
the problem is that having no print data is considered as an illegal state for the report engine: https://github.com/idempiere/idempiere/blob/master/org.adempiere.base/src/org/compiere/print/ReportEngine.java#L391

Reply:
that's implementation detail, logically, it doesn't make any sense to call loadPrintData for jasper report.

Reply:
Before fixing this issue, what do you think about [this suggestion](https://idempiere.atlassian.net/browse/IDEMPIERE-5973?focusedCommentId=50440)?

Reply:
If you can make it work, that sounds fine to me.